packages/office-ui-fabric-react/src/components/Pivot/Pivot.scss

```diff
@@ -82,6 +82,11 @@
   &.is-disabled {
     color: $ms-color-neutralTertiary;
   }
+  
+  // Spacing between text and count 
+  .ms-Pivot-count {
+    @include margin-left(4px);
```

Comment:
Since the code was only moved, I'll keep it with the mixin for now, and open a new issue for the LTR support.